content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -87,51 +139,54 @@ To retrieve data from Filecoin using Lassie, all you need is the CID of the cont
     ...
     ```
 
-1. This will create an `output.car` file within your current directory:
+* `-o` is an optional flag that tells Lassie where to write the output to. If you don't specify a file, it will append `.car` to your CID and use that as the output file name.
 
-    ```shell
-    ls -l
-    ```
-
-    ```shell
-    total 143M
-    -rw-rw-r-- 1 user user 143M Feb 16 11:21 output.car
-    ```
+  If you specify `-`, as in our above example, the output will be written to stdout so it can be piped to another command, such as go-car, or redirected to a file.
 
-### Extract data
+* `<CID>/path/to/content` is the CID of the content you want to retrieve, and an optional path to a specific file within that content.
 
-Now that we’ve downloaded a CAR file, we need to find out what’s inside it. 
+  A CID is always necessary and if you don't specify a path, Lassie will attempt to download the entire content. If you specify a path, Lassie will only download that specific file, or if it is a directory, the entire directory and its contents.
 
-1. The format for extracting a `.car` file using Go-car is:
+##### go-car CLI usage
 
-    ```shell
-    car extract --file <INPUT_FILE>
-    ```
+The `car extract` command can be used to extract files and directories from a CAR:
```

Comment:
perhaps note about where to download and install go-car?

Reply:
This is already there at https://github.com/filecoin-project/filecoin-docs/pull/1812/files#diff-392c257e64c55c339f84cd9e5759891b48359815e5fd0fbf55bbefeb6c025e19R53 This is a subsection and we are already installing binaries in the beginning of this section.